src/types/ErrorType.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorType {
+  none = '',
```

Comment:
enum keys should be capitalized, it's the best practice
```suggestion
  None = '',
```